2016/proposals/verma-nikhil.md

```diff
@@ -0,0 +1,232 @@
+
+* Implement social authentication
+* Implement the messaging system
+* Write tests for application form and Trainee Dashboard
+
+### July 20th - August 2nd
+
+* Write tests for Trainee Admin
+* First draft ready for review
+* Review *admin*'s feedback
+
+### August 3rd - August 16th
+
+* Work on reviews from *admins*
+* Final optimizations
+* Wind up writing tests
+* Find and fix critical bugs
+* Write documentation
+
+### August 17th - August 21th 19:00 UTC
+
+Buffer for final wrap up, fix if something remains.
+
+## Future works
+
+The system has enormous scope of expansion, the major ones being:
+
+1. Integrate discussion session within the portal, which is like a Google Doc open within the portal
+2. Incorporate video calling with screen sharing abilities from within the portal to integrate checkout sessions.
+
+
+## Open Source Development Experience
+
+  As far as 'Open Source' is concerned, I've only contributed to AMY and related projects. I have:
+
+  * [reported a bug](https://github.com/swcarpentry/amy/pull/720), which returned all persons instead of none if no duplicates were found, and also provided a [fix](https://github.com/swcarpentry/amy/pull/720) for it, which has been merged.
+  * working on [expanding AMY to incorporate certificate generation](https://github.com/swcarpentry/amy/pull/724). The pull request is under discussion
+  * helped [eliminate the use of Inkscape](https://github.com/swcarpentry/certification/pull/12) by replacing it with CairoSVG in the certification repository
+  * helped [tweak some styles](https://github.com/gvwilson/python-novice-gapminder/pull/1) in the [python-novice-gapminder](https://github.com/swcarpentry/python-novice-gapminder) repository
+
+## Academic Experience
+
+I am an undergraduate from BITS Pilani pursuing majors in Electrical and Electronics Engineering at BITS Pilani. I'm currently in my second year and have taken a variety of interesting courses such as Control Systems, Microprocessors Programming, Digital Design, OOP and Machine Learning.
+I'm also interested in web development, and I develop websites of our college fests. Some projects undertaken during college are:
+* The backend of websites of our college fest, built on Django: [Oasis](http://bits-oasis.org/), [BOSM](http://bits-bosm.org) and [APOGEE](http://bits-apogee.org)
+* An event management system, to keep track of the progress of teams in an event as a part of APOGEE 2016
+* An in house registrations portal, to assign IDs, billing and allot accomodation to outstation participants
+
+## Why this project?
+
+I've developed Django applications in the past, and I wanted to improve upon my skills by means of working with experienced developers for some real world applications. And after some contributions to AMY, I seem to understand it well, and I'm also comfortable enough with AMY. This gives me more reasons to work on it! Also, I can promise quality work, and hope that this becomes a wonderful experience for all of us.
```

Comment:
All of them! Apogee 16 is fully based on Django, while Oasis and BOSM use a django backend connected to the frontend via AJAX. I've added live previews and GitHub links for all of them.